support-frontend/assets/pages/subscriptions-redemption/subscriptionsRedemption.jsx

```diff
@@ -52,7 +52,7 @@ const content = (
     >
       <CheckoutStage
         checkoutForm={<RedemptionForm />}
-        thankYouContentPending={<ThankYouPendingContent {...thankyouProps} />}
+        thankYouContentPending={<ThankYouPendingContent includePaymentCopy={false} {...thankyouProps} />}
```

Comment:
You could add `includePaymentCopy: false` to the `thankyouProps` object and save having to put an inline prop here.